src/main/kotlin/org/jetbrains/research/transformations/Transformations.kt

```diff
@@ -0,0 +1,36 @@
+package org.jetbrains.research.transformations
+
+import com.github.gumtreediff.tree.ITree
+
+public class Transformations() {
```

Comment:
@nbirillo is totally right, but I want to argue with names (as usual, haha). Let it be either
```
 interface Transformable {
    fun transform(tree: ITree, toStoreMetadata : Boolean)
    fun reverseTransform(tree: ITree)
}
```
or 

```
interface Transformation {
    fun apply(tree: ITree, toStoreMetadata : Boolean)
    fun reverseApply(tree: ITree)
}
```
because you can **apply a transformation** or **transform something transformable**.

Reply:
Thanks! It is a good idea!